packages/types/src/extensions/retry.ts

```diff
@@ -0,0 +1,10 @@
+import { RetryStrategyV2 } from "../retry";
+import { Provider, RetryStrategy } from "../util";
+
+/**
+ * @internal
+ */
+export interface RetryStrategyConfiguration {
+  setRetryStrategy(retryStrategy: Provider<RetryStrategyV2 | RetryStrategy>): void;
```

Comment:
add a jsdoc to this method: what situation involves calling this, and what is the expected consequence?